### DISCUSSION TITLE
Why there is no proper Redis Client Documentation for Node.js?

### DISCUSSION THREAD

**Discussion by patrickcavli** [NodeRedis/node_redis#2577] [Q&A]
Dear Redis Team,

There is a ton of Redis documentation fragmented out there, as a Developer. I'm struggling to find what all are the function signatures and parameters for various Redis operation. If there is some kind of documentation out there other than CLI one. Please let me know.

**Comment by vpzomtrrfrt** [comment]
there's this: https://redis.js.org/documentation/client/

but it's not really convenient to look at

**Comment by bhuffman** [comment]
This is beyond not convenient, it is actually pretty useless for basic questions. For example, where in those docs does it say that the syntax for "SADD" in the node redis package is sAdd and not sadd (like it used to be) or SADD? We DESPERATELY need documentation like this: https://ramdajs.com/docs/
